Path_generator:
  ros__parameters:
    x_goal: 1.0
    y_goal: 1.0
